### PR TITLE
doc: Add Conjure setup to development.md

### PR DESCRIPTION
**Pull request by ssjoleary**

### CODE REVIEW

**Comment by joelhans** [comment]
Hey @ssjoleary—thanks for submitting the PR.

All of the files in this repo have moved around since you submitted this PR, hence the merge conflict. The file you initially edited doesn't exist any more. It's now at `/company/athens-research-governance-and-operations/engineering/development.md`. Sorry for the inconvenience!

Looks like GitHub offers some tips on merging, or you can try rebasing your PR based on the latest `main` and adding your changes into the new file. Let me know if you have questions! Then I can help push this through.

**Comment by ssjoleary** [comment]
No problem @joelhans, thanks for the heads up!

File: company/athens-research-governance-and-operations/engineering/development.md [hunk 1]
```diff
@@ -156,6 +156,66 @@ You supposed to see an alert on electron app saying "hello!" and your Vim instan
 
 If all goes well, now you can see documentation of symbols \(binding: K\), go to definition \(binding: \[ C-d\) and so fort. See `:help fireplace` for more information.
 
+#### Conjure
+
+[Conjure](https://github.com/Olical/conjure) is an interactive environment for evaluating code within your running program for Neovim.
+[Installing Conjure](https://github.com/Olical/conjure#installation)
+
+It's main dependency is the [cider-nrepl](https://github.com/clojure-emacs/cider-nrepl).
```
Comment (joelhans) [inline company/athens-research-governance-and-operations/engineering/development.md:164]:
```suggestion
Its main dependency is the [cider-nrepl](https://github.com/clojure-emacs/cider-nrepl).
```

File: company/athens-research-governance-and-operations/engineering/development.md [hunk 2]
```diff
@@ -156,6 +156,66 @@ You supposed to see an alert on electron app saying "hello!" and your Vim instan
...
+Create the following file at `~/.shadow-cljs/config.edn` to add the dependency;
```
Comment (joelhans) [inline company/athens-research-governance-and-operations/engineering/development.md:165]:
```suggestion
Create the following file at `~/.shadow-cljs/config.edn` to add the dependency:
```

File: company/athens-research-governance-and-operations/engineering/development.md [hunk 3]
```diff
@@ -156,6 +156,66 @@ You supposed to see an alert on electron app saying "hello!" and your Vim instan
...
+
+```text
+lein dev # in one terminal, running nrepl server on port 8777
+yarn run electron . # another terminal running the Athens app itself
+```
+
+Open any Clojurescript file in Neovim.
+Conjure will connect automatically to the leiningen nrepl server and you should see a temporary floating buffer in the top right corner of your editor showing a successful connection.
+
+To get that buffer back and to keep it open in a horizontal split;
```
Comment (joelhans) [inline company/athens-research-governance-and-operations/engineering/development.md:185]:
```suggestion
To get that buffer back and to keep it open in a horizontal split:
```

**Review by joelhans — CHANGES_REQUESTED** [review]
Functionally this LGTM! I don't know much about this process or what the value is for developers, so pinging @neotyk or @tangjeff0 in case they want to weigh in.

I'll give them some time, and if there's no more feedback, we'll go ahead and merge.

**Comment by tangjeff0** [comment]
This value is that it's another dev environment for another text editor - Vim in this case. So pretty valuable!

**Comment by joelhans** [comment]
> This value is that it's another dev environment for another text editor - Vim in this case. So pretty valuable!

Awesome—didn't realize there were separate environments for different editors. We'll push this as soon as @ssjoleary has time to look at some of the major changes I suggested.

**Review by joelhans — APPROVED** [review]
Thanks very much @ssjoleary! Approving and merging now.

**Pull request merged**